locksmith/__tests__/controllers/v2/ticketsController.test.ts

```diff
@@ -78,4 +103,44 @@ describe('sign endpoint', () => {
 
     expect(response.status).toBe(202)
   })
+
+  it('correctly marks ticket as checked-in without metadata', async () => {
+    expect.assertions(3)
+    const { loginResponse } = await loginRandomUser(app)
+    expect(loginResponse.status).toBe(200)
+
+    const metadata = {}
+    const response = await request(app)
+      .put(`/v2/api/ticket/${network}/lock/${lockAddress}/key/${tokenId}/check`)
+      .set('authorization', `Bearer ${loginResponse.body.accessToken}`)
+      .send(metadata)
+    expect(response.status).toBe(202)
+
+    const updateStatus = await metadataOperations.updateKeyMetadata(metadata)
+    expect(updateStatus).toBe(false)
+  })
+
+  it('correctly marks ticket as checked-in and updates metadata', async () => {
```

Comment:
This test should set metadata first, then check the ticket in and read the metadata to verify it is correct